server/chat-commands/info.ts

```diff
@@ -1539,7 +1539,8 @@ export const commands: Chat.ChatCommands = {
 		const globalRanks = [
 			`<strong>Global ranks</strong>`,
 			`+ <strong>Global Voice</strong> - They can use ! commands like !groups`,
-			`% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts`,
+			`§ <strong>Section Leader</strong> - They oversee rooms in a particular section`,
+			`% <strong>Global Driver</strong> - Like Voice, and they can lock users and check for alts`,
```

Comment:
You'll need to update the Driver translation texts too.

Reply:
Agh, okay. I'll post in wail about it.